### ISSUE TITLE
Early termination of training with YALTAi

### ISSUE THREAD

**Issue by PaulineJac** [FoNDUE-HTR/Documentation#17]
Dear Fondue Team 🫕! 

I have had a problem training segmentation models with Yaltai for the past week.
I use this command to convert my alto data to yolo:
`yaltai alto-to-yolo /home/users/j/jacsont/Varios-OCR-files/Grountruth/Data+varios/*.xml Yaltai_varios_1 --shuffle .1 --segmonto subtype`
The repository with the data is [here](https://github.com/DesenrollandoElCordel/FoNDUE-Spanish-chapbooks-Dataset/tree/main/Grountruth/Data%2Bvarios).
Then I use these commands to start a training: 
`salloc --partition=public-gpu --time=02:00:00 --gpus=1`
`module load fosscuda/2020b Python/3.8.6`
`source ~/Experimentum-env/bin/activate`
`python train.py --data "../Yaltai_varios_1/config.yml" --batch-size 4 --img 640 --weights yolov5x.pt --epochs 50`

But, the training starts and stops quickly without giving any error: 
````
train: weights=yolov5x.pt, cfg=, data=../Yaltai_varios_2/config.yml, hyp=data/hyps/hyp.scratch-low.yaml, epochs=50, batch_size=4, imgsz=640, rect=False, resume=False, nosave=False, noval=False, noautoanchor=False, noplots=False, evolve=None, bucket=, cache=None, image_weights=False, device=, multi_scale=False, single_cls=False, optimizer=SGD, sync_bn=False, workers=8, project=runs/train, name=exp, exist_ok=False, quad=False, cos_lr=False, label_smoothing=0.0, patience=100, freeze=[0], save_period=-1, seed=0, local_rank=-1, entity=None, upload_dataset=False, bbox_interval=-1, artifact_alias=latest
github: skipping check (offline), for updates see https://github.com/ultralytics/yolov5
Unknown option: -C
usage: git [--version] [--help] [-c name=value]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p|--paginate|--no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           <command> [<args>]
YOLOv5 🚀 2022-11-21 Python-3.8.6 torch-1.11.0+cu102 CUDA:0 (Tesla V100-PCIE-32GB, 32511MiB)

hyperparameters: lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=0.05, cls=0.5, cls_pw=1.0, obj=1.0, obj_pw=1.0, iou_t=0.2, anchor_t=4.0, fl_gamma=0.0, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0
Weights & Biases: run 'pip install wandb' to automatically track and visualize YOLOv5 🚀 runs in Weights & Biases
ClearML: run 'pip install clearml' to automatically track, visualize and remotely train YOLOv5 🚀 in ClearML
TensorBoard: Start with 'tensorboard --logdir runs/train', view at http://localhost:6006/
Overriding model.yaml nc=80 with nc=13

                 from  n    params  module                                  arguments                     
  0                -1  1      8800  models.common.Conv                      [3, 80, 6, 2, 2]              
  1                -1  1    115520  models.common.Conv                      [80, 160, 3, 2]               
  2                -1  4    309120  models.common.C3                        [160, 160, 4]                 
  3                -1  1    461440  models.common.Conv                      [160, 320, 3, 2]              
  4                -1  8   2259200  models.common.C3                        [320, 320, 8]                 
  5                -1  1   1844480  models.common.Conv                      [320, 640, 3, 2]              
  6                -1 12  13125120  models.common.C3                        [640, 640, 12]                
  7                -1  1   7375360  models.common.Conv                      [640, 1280, 3, 2]             
  8                -1  4  19676160  models.common.C3                        [1280, 1280, 4]               
  9                -1  1   4099840  models.common.SPPF                      [1280, 1280, 5]               
 10                -1  1    820480  models.common.Conv                      [1280, 640, 1, 1]             
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 12           [-1, 6]  1         0  models.common.Concat                    [1]                           
 13                -1  4   5332480  models.common.C3                        [1280, 640, 4, False]         
 14                -1  1    205440  models.common.Conv                      [640, 320, 1, 1]              
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 16           [-1, 4]  1         0  models.common.Concat                    [1]                           
 17                -1  4   1335040  models.common.C3                        [640, 320, 4, False]          
 18                -1  1    922240  models.common.Conv                      [320, 320, 3, 2]              
 19          [-1, 14]  1         0  models.common.Concat                    [1]                           
 20                -1  4   4922880  models.common.C3                        [640, 640, 4, False]          
 21                -1  1   3687680  models.common.Conv                      [640, 640, 3, 2]              
 22          [-1, 10]  1         0  models.common.Concat                    [1]                           
 23                -1  4  19676160  models.common.C3                        [1280, 1280, 4, False]        
 24      [17, 20, 23]  1    121122  models.yolo.Detect                      [13, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [320, 640, 1280]]
Model summary: 567 layers, 86298562 parameters, 86298562 gradients, 204.9 GFLOPs

Processus arrêté
````

I don't know what the problem is and I thank you for your help.

**Comment by PaulineJac** [comment]
Thanks to @gabays, the problem has been solved. 

The process stops due to a RAM shortage on my CPU. 
Following  Simon Gabay's advice, I used this command to ask for the GPU: 
`salloc --partition=shared-gpu --time=01:00:00 --gpus=1 --mem=24GB --cpus-per-task=8 --gres=gpu:1,VramPerGpu:24G` 

Then I trained my Yalatai segmentation model using this command: 
`python train.py --data "../Yaltai_varios_2/config.yml" --batch-size 4 --img 640 --weights yolov5x.pt --epochs 50 --workers 8`

**Comment by gabays** [comment]
If you can try with
```console
python train.py --data "../Yaltai_varios_2/config.yml" --batch-size 4 --img 1280 --weights yolov5x6.pt --epochs 50 --workers 8
```
and mybe compare the results with the previous training, it would be interesting -- and you might improve your results
